bundles/binding/org.openhab.binding.zwave/database/homeseer/ezmultipli.xml

```diff
@@ -0,0 +1,117 @@
+20 min for an office environment<BR/>
+60 min for a library, office or other room where someone may be sitting still for a long time.
+			]]></Help>
+		</Parameter>
+
+		<Parameter>
+			<Index>2</Index>
+			<Type>int</Type>
+			<Minimum>-1</Minimum>
+			<Maximum>99</Maximum>
+			<Default>-1</Default>
+			<Size>1</Size>
+			<Label lang="en">OnLevel</Label>
+			<Help lang="en"><![CDATA[
+OnLevel is the value sent by the Z-Wave BASIC_SET command to all Association Group 2 nodes when motion is detected.<BR/>
+A value of 0 will turn the lights off (not recommended).<BR/>
+A value between 1 and 99 will set the dim level to between 1% and 99% (99% is full on).<BR/>
+A value of -1 will turn the light “on” which depends on the device but most will set the dim level to the last dim setting.<BR/>
+			]]></Help>
+		</Parameter>
+
+		<Parameter>
+			<Index>3</Index>
+			<Type>int</Type>
+			<Minimum>0</Minimum>
+			<Maximum>127</Maximum>
+			<Default>60</Default>
+			<Size>1</Size>
+			<Label lang="en">LiteMin</Label>
+			<Help lang="en"><![CDATA[
+A Luminance report is sent to the system controller every LiteMin minutes.<BR/>
+A value of zero turns this mode off. Luminance values can still be obtained at any time by the home control application in this mode.
+			]]></Help>
+		</Parameter>
+		
+		<Parameter>
+			<Index>4</Index>
+			<Type>int</Type>
+			<Minimum>0</Minimum>
+			<Maximum>127</Maximum>
+			<Default>60</Default>
+			<Size>1</Size>
+			<Label lang="en">TempMin</Label>
+			<Help lang="en"><![CDATA[
+A Temperature report is sent to the controller every TempMin minutes.<BR/>
+A value of zero turns this mode off. Temperature values can still be obtained at any time by polling the status of the temperature sensor.
+			]]></Help>
+		</Parameter>
+
+		<Parameter>
+			<Index>5</Index>
+			<Type>int</Type>
+			<Minimum>-127</Minimum>
+			<Maximum>128</Maximum>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">TempAdj</Label>
+			<Help lang="en"><![CDATA[
+TempAdj is a twos-complement number that is used to adjust the temperature reading to make it more accurate. The value is in tenths of degree Fahrenheit. The temperature reading can be adjusted by +12.7F to -12.8F. A value of 1 will adjust the temperature reading by +0.1F. A value of -1 will adjust the temperature by -0.1F. A value of 123 will adjust the temperature by +12.3F. 
+			]]></Help>
+		</Parameter>
+	</Configuration>
+
+	<Associations>
+		<Group>
+			<Index>1</Index>
+			<Maximum>1</Maximum>
+			<Label lang="en">LIFELINE</Label>
+			<Help lang="en"><![CDATA[
+This group is normally automatically assigned to the Z-Wave system controller when EZMultiPli is added to the ZWave network. All Notification command class reports (motion detection) are sent to the node assigned to this group. Advanced lighting control should be handled by a home control application based on these reports.			
+			]]></Help>
+		</Group>
+		<Group>
+			<Index>2</Index>
+			<Maximum>4</Maximum>
```

Comment:
If the lifeline doesn't also get reports when the light comes on, then this shoudl also have the SetToController true